tests/handlers/test_presence.py

```diff
@@ -725,13 +731,6 @@ def test_set_presence_from_syncing_keeps_status(self) -> None:
         self.assertEqual(state.status_msg, status_msg)
 
     @parameterized.expand([(False,), (True,)])
-    @unittest.override_config(
-        {
-            "experimental_features": {
-                "msc3026_enabled": True,
-            },
-        }
-    )
```

Comment:
Do we want a separate test for the config vs. the per-user? (Same question for the other tests too.)

Reply:
Yeah that probably makes sense - I've added tests for config and per-user and made sure the per-user tests were activated through the api vs the db for more completeness.